cmake/FetchCIMpp.cmake

```diff
@@ -8,26 +6,17 @@ message(STATUS "CIM Version: ${CIM_VERSION}")
 
 if(CIM_VERSION STREQUAL "16v29a")
 	set(USE_CIM_VERSION "IEC61970_16v29a")
-endif()
-if(CIM_VERSION STREQUAL "CGMES_2.4.15_16FEB2016")
+elseif(CIM_VERSION STREQUAL "CGMES_2.4.15_16FEB2016")
 	set(USE_CIM_VERSION "CGMES_2.4.15_16FEB2016")
-  set(CGMES_BUILD ON)
+	set(CGMES_BUILD ON)
 endif()
 
 include(FetchContent)
-FetchContent_Declare(
-  cimpp-module
-  GIT_REPOSITORY https://github.com/cim-iec/libcimpp.git
-  GIT_TAG        master
+FetchContent_Declare(cimpp-module
+	GIT_REPOSITORY https://github.com/cim-iec/libcimpp.git
```

Comment:
Because its redundant. FetchContent will fetch the default branch which is currently `master`.
The old approach will fail if somebody renames the default branch.